lib/member_page.rb

```diff
@@ -36,6 +40,14 @@ class MemberPage < Scraped::HTML
 
   private
 
+  def titles
+    %w(Dr)
+  end
+
+  def name_parts
+    box.css('h1').text.split(' - ').first.sub('Hon. ', '').sub(' MP', '').tidy.split(' ')
```

Comment:
now that you're splitting the name up and putting it back together again anyway, it doesn't really make a lot of sense to manipulate the raw text like this. I'd suggest dropping that part of this, setting up separate lists, along the lines of:

```ruby
  def wanted_titles
    %w(Dr)
  end

  def unwanted_titles
    %w(Hon. MP)
  end

  def titles
    wanted_titles + unwanted_titles
  end
```

and then changing `honorifc_prefix` to only include parts in `wanted_titles`